docs/docs/administer/regen-keys.mdx

```diff
@@ -10,7 +10,7 @@ If you're looking to regenerate one or more of these secrets, it's possible
 to do so using the same bootstrap scripts that were used when installing
 Firezone.
 
-## Regenerate Secrets
+## Regenerate secrets
```

Comment:
Just confirming -- are we establishing this capitalization scheme as a convention? Level 1 headings are capitalized, Level 2 and below only the first word capitalized?

Reply:
I do like sentence case better (even for lv1 headers). It feels more casual and conversational. It also seems easier to write titles that are more descriptive.

 I made all the headers in sentence case in the PR (unless I missed something).

Reply:
I did not go back to all the previously written docs to update this. I should once we decide on a format.